### ISSUE TITLE
Show quantity as well as price

### ISSUE THREAD

**Issue by Pandemonious** [myzticbean/QSFindItemAddOn#37]
Hello,

One issue I find with using this mod arises when setting "allow-stacks: true" in Quickshop Hikari or Reremake. Without a quantity to go along with the price, is impossible to know the actual rate per item, and thus can't compare prices within the GUI.

Specifically, I'm looking at this section of the config:
```
# Shop Search GUI Item Lore
# Placeholders for shop-item-lore:
# {ITEM_PRICE}          -> Item price in shop
# {SHOP_STOCK}          -> Remaining stock in shop
# {SHOP_OWNER}          -> Owner of shop
# {SHOP_LOC}            -> Shop coordinates
# {SHOP_WORLD}          -> Name of the world the shop is in
# {NEAREST_WARP}        -> Warp/Region nearest to shop
# {SHOP_VISITS}         -> Total Player visits to shop
shop-gui-item-lore:
- ''
- '&fPrice: &a{ITEM_PRICE}'
- '&fStock: &7{SHOP_STOCK}'
- '&fOwner: &7{SHOP_OWNER}'
- '&fLocation: &7{SHOP_LOC}'
- '&fWorld: &7{SHOP_WORLD}'
- '&fWarp: &7{NEAREST_WARP}'
- '&fShop Visits: &7{SHOP_VISITS}'
- ''
```

In my humble opinion, a new placeholder is necessary here, maybe call it {ITEM_QTY}. In the item lore, it could simply be on line 2 with the price. Ex: &fPrice: &a${ITEM_PRICE} &rper &a{ITEM_QTY}

Thank you for reading my suggestion.
Best regards,

**Comment by myzticbean** [comment]
@Pandemonious This is a good suggestion. Would you rather me show it as a stack size in the gui? Something like this?
![image](https://github.com/myzticbean/QSFindItemAddOn/assets/86056736/6f3b19da-20ac-4d18-aebb-a22cfb8d2eea)

**Comment by Pandemonious** [comment]
You've made the idea even better. That is an excellent option to have as well. Thanks - looking forward to release.

**Comment by myzticbean** [comment]
Available in v2.0.6.3, closing issue.
